testapp/src/main/res/layout/qa_settings_activity.xml

```diff
@@ -247,6 +247,47 @@
                 app:pageName="QA"
                 app:siteSection="Settings" />
 
+            <androidx.appcompat.widget.AppCompatTextView
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:background="@color/bg_section"
+                android:paddingStart="@dimen/medium_16"
+                android:paddingTop="@dimen/small_4"
+                android:paddingEnd="@dimen/medium_16"
+                android:paddingBottom="@dimen/small_4"
+                android:text="@string/universal_bridge" />
+
+            <com.google.android.material.textfield.TextInputLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:padding="@dimen/small_4"
+                app:endIconMode="clear_text">
+
+                <androidx.appcompat.widget.AppCompatEditText
+                    android:id="@+id/edtUniversalBridgeMessage"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginBottom="@dimen/small_4"
+                    android:hint="@string/hint_universal_bridge_message"
+                    android:imeOptions="actionDone"
+                    android:inputType="text" />
```

Comment:
We can change to `textMultiLine` to input a json like this:

![Screen Shot 2022-12-02 at 12 36 30](https://user-images.githubusercontent.com/66930373/205209456-c108428f-87aa-4cab-95d9-db5c7c7bfa22.png)

```suggestion
                    android:inputType="textMultiLine" />
```